ci_scripts/check_image_format.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
```

Comment:
`shellcheck` spits out a ton of warnings and suggestions for this script. Would be good to go in and clean them up

Reply:
Hmm I wasn't aware of shellcheck. I'll give it a shot. thanks!